server/internal/webrtc/webrtc.go

```diff
@@ -211,6 +213,25 @@ func (manager *WebRTCManager) CreatePeer(id string, session types.Session) (stri
 		return "", manager.config.ICELite, manager.config.ICEServers, err
 	}
 
+	go func() {
+		rtcpBuf := make([]byte, 1500)
+		for {
+			if _, _, rtcpErr := rtpVideo.Read(rtcpBuf); rtcpErr != nil {
+				return
+			}
+		}
+	}()
+
+	go func() {
+		rtcpBuf := make([]byte, 1500)
+		for {
+			if _, _, rtcpErr := rtpAudio.Read(rtcpBuf); rtcpErr != nil {
+				return
+			}
+		}
+	}()
```

Comment:
I am skeptical about adding two new gorutines for each participants. Not sure, if it is really adding some value, it would need more testing, I guess. I see, that it is used in lots of examples, but is that not just some sort of a test? It is reading all video data for the client as well? Or are those data something different than the samples?

Reply:
As far as I am understanding it, it is listening for small responses from the clients, that could send NACKs and congestion control messages. Boths are things that I _really_ want to have in Neko, especially, when it comes out of the box.

I think the pion made the decision, that interceptors will only listen to given Read and Write and will not create its one goroutines.

Reply:
Okey, I just needed to be sure, that they end when connection dies. And that is happening, so LGTM.